config/environments/development.rb

```diff
@@ -14,8 +14,15 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { host: 'rubygamedev.dev' }
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_url_options = { host: "localhost:3000" }
+  config.action_mailer.delivery_method = :smtp
```

Comment:
The development environment should not send email. The latest version of Rails includes a way of previewing the emails without having to send them.